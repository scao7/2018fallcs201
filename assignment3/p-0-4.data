11 2 467 ; 12 13 217 ; 0 10 160 ; 1 3 359 ; 14 11 323 ; 
7 0 232 ; 10 1 438 ; 12 10 477 ; 9 1 345 ; 3 0 280 ; 
6 8 306 ; 3 5 78 ; 13 1 50 ; 0 12 166 ; 12 12 172 ; 
12 8 278 ; 14 14 141 ; 8 15 291 ; 9 0 143 ; 9 0 448 ; 
1 7 204 ; 7 9 170 ; 8 7 61 ; 5 5 394 ; 1 5 130 ; 
9 4 146 ; 7 9 363 ; 6 9 281 ; 7 4 273 ; 7 9 198 ; 
14 1 51 ; 7 15 153 ; 
