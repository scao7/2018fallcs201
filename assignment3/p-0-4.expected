0: 11
1: 2(11)467 14(11)323
2: 1(14)51
3: 5(1)130 7(1)204 13(1)50
4: 3(5)78 8(7)61 9(7)170 15(7)153
5: 0(9)143 4(9)146 6(9)281
6: 10(0)160 12(0)166
weight: 2583
